src/index.tsx

```diff
@@ -1,15 +1,28 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
-import { createStore } from "redux";
+import { createStore, StoreEnhancer } from "redux";
 import * as Sentry from "@sentry/react";
 import { Integrations } from "@sentry/tracing";
 import App from "./app";
 import "./assets/styles/styles-all.scss";
 import * as serviceWorker from "./serviceWorker";
 import { appReducer } from "./state/app-reducer";
 
-const appStore = createStore(appReducer);
+type WindowWithDevTools = Window & {
+  __REDUX_DEVTOOLS_EXTENSION__: () => StoreEnhancer<unknown, {}>;
+};
+
+const isReduxDevtoolsExtAvailable = (
+  arg: Window | WindowWithDevTools
+): arg is WindowWithDevTools => {
+  return "__REDUX_DEVTOOLS_EXTENSION__" in arg;
+};
+
+const appStore = createStore(
+  appReducer,
+  isReduxDevtoolsExtAvailable(window) ? window.__REDUX_DEVTOOLS_EXTENSION__() : undefined
```

Comment:
To co dodałem w tym pliku umożliwia funkcjonowanie wtyczki Redux DevTools, która pomaga przy debugowaniu stanu - https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd

Reply:
Rozwiązanie zaczerpnięte z dokumentacji: https://github.com/zalmoxisus/redux-devtools-extension#user-content-11-basic-store:~:text=For%20TypeScript%20use%20redux%2Ddevtools%2Dextension%20npm%20package%2C,all%20the%20definitions%2C%20or%20just%20use